### PR TITLE
Add clarifications to gce getting started guide.

### PR DESCRIPTION
**Pull request by rjnagal**
Based on last week's UX study:

. Mention that instance groups API needs to be enabled.
. Add link to cloud SDK.
. Add more info on cluster addon that show up on bringup.

### CODE REVIEW

**Comment by rjnagal** [comment]
cc @vmarmol

File: docs/getting-started-guides/gce.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The next few steps will show you:
 
 ### Installing the kubernetes client on your workstation
```
Comment (gmarek) [inline docs/getting-started-guides/gce.md]: I don't like `kubernetes client` phrase. I'd rather have `kubernetes command line tools` or something which does not collide with internal `client` library.

**Comment by vmarmol** [comment]
LGTM

+1 to @gmarek's suggestion.

**Comment by rjnagal** [comment]
@gmarek updated. Thanks.

**Comment by vmarmol** [comment]
LGTM, thanks @rjnagal! Will merge on green since it is a doc-only change.

**Pull request merged**